controllers/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -524,7 +535,7 @@ func (r *Reconciler) sendToolchainStatusNotification(logger logr.Logger,
 }
 
 func ClusterURLs(instance *toolchainv1alpha1.ToolchainStatus) map[string]string {
-	cfg, err := rest.InClusterConfig()
+	_, err := rest.InClusterConfig()
```

Comment:
if we don't use `cfg` anymore should we avoid invoking this function?

Reply:
The reason i still have this call is because there is a logic to populate the map with `Member Cluster`, if there is an errir to retrieve InClusterConfig. Just wanted to retain that logic. https://github.com/codeready-toolchain/host-operator/blob/master/controllers/toolchainstatus/toolchainstatus_controller.go

Reply:
Right, but since we now want to use `toolchainStatus.Status.HostRoutes.ProxyURL` instead of `cfg` maybe we could check if `toolchainStatus.Status.HostRoutes.ProxyURL` is set here instead and if we can parse it using `removeSchemeFromUrl`. If not then we fallback on the logic that populates the map with `Member Cluster` logic. WDYT?

Reply:
Yeah that can de done.